app/_src/guides/consumer-producer-policies.md

```diff
@@ -0,0 +1,231 @@
+kubectl get meshtimeout -n kuma-demo producer-timeout -o jsonpath='{.metadata.labels}'
+```
+
+We should see output like:
+
+```json
+{
+  "k8s.kuma.io/namespace": "kuma-demo",
+  "kuma.io/env": "kubernetes",
+  "kuma.io/mesh": "default",
+  "kuma.io/origin": "zone",
+  "kuma.io/policy-role": "producer",
+  "kuma.io/zone": "default"
+}
+```
+
+{{site.mesh_product_name}} is adding custom labels to our policy. The one that interests us most at the moment is `kuma.io/policy-role`.
+This is new label that indicates policy role. Possible values of this label are:
+
+- `system` - policies applied in system namespace
+- `workload-owner` - policies that don't specify `spec.from` and `spec.to` sections or specify only `spec.from` section
+- `consumer` - policies targeting MeshServices from different namespace or targeting MeshService by labels
+- `producer` - policies targeting MeshServices from the same namespace in `spec.to[]` section 
+
+### Producer consumer model
+
+With namespace scoped policies we've introduced producer/consumer model for policies. 
+
+**Producer** is someone who authors and operates service. Producer can create default policies that will be used when communicating with service. 
+Producer policies will be created in the same zone as MeshService they target. Producer policies will be synced to other zones. 
+
+**Consumer** is someone that utilizes a service by communicating with it. Consumer policies will be applied in consumer 
+namespace and will target MeshService from different namespace. Consumer policy will take effect only in consumer namespace.
+Consumer policies will override producer policies.
+
+## Testing producer policy
+
+To test MeshTimeout that we've applied in previous steps we need to simulate delays on our requests. To do this we need
+add header `x-set-response-delay-ms` to our requests
+
+We can now make few requests to our demo-app, and we should see timeouts:
+
+```shell
+kubectl exec -n first-consumer consumer -- curl -s -XPOST demo-app.kuma-demo:5000/increment -H "x-set-response-delay-ms: 2000"
+```
+
+Example output:
+
+```
+{"counter":"3","zone":"local","err":null}
+```
+
+We should see the same results when making requests from second-consumer namespace:
+
+```shell
+kubectl exec -n second-consumer consumer -- curl -s -XPOST demo-app.kuma-demo:5000/increment -H "x-set-response-delay-ms: 2000"
+```
+
+## Utilizing consumer policy
+
+To this moment we were relying on producer policy created previously. Let's assume that we don't mind waiting a little longer
+for the response from demo-app, and we would like to override the timeout just for our namespace. To do so we need to create new policy:
+
+```shell
+echo "apiVersion: kuma.io/v1alpha1
+kind: MeshTimeout
+metadata:
+  name: consumer-timeout
+  namespace: first-consumer
+  labels:
+    kuma.io/mesh: default
+    kuma.io/origin: zone
+spec:
+  to:
+    - targetRef:
+        kind: MeshService
+        name: demo-app
+        namespace: kuma-demo
```

Comment:
Same re `labels` it makes things a bit more obvious IMO

Reply:
changed